src/Serialization/Fields.jl

```diff
@@ -1,3 +1,36 @@
+################################################################################
+# Utility functions for parent tree
+function get_parents(parent_ring::Field)
+    if absolute_degree(parent_ring) == 1
+        return Any[]
+    end
+    base = parent(defining_polynomial(parent_ring))
+    parents = get_parents(base)
+    push!(parents, parent_ring)
+    return parents
+end
+
+function get_parents(parent_ring::T) where T <: Union{NfAbsNS, NfRelNS}
+    ngens = length(gens(parent_ring))
```

Comment:
Strictly speaking instead of `length(gens(x))` one should do `ngens(x)`, *but* (a) I have not verified it exists in those case (I mean: it *should* but it might be one of the many places that we need to fix...); and (b) of course then the local variable shouldn't be called `ngens` anymore but e.g. `n`